release notes/v0.36.0.md

```diff
@@ -0,0 +1,108 @@
+k6 v0.36.0 is here! 🎉 It introduces a couple new features which enhance its usability, and includes a bunch of fixes and the result of refactoring ongoing efforts.
+
+In total, we have closed 27 issues.
+
+## New Features!
+
+### Sourcemaps support ([#2082](https://github.com/grafana/k6/pull/2082))
+
+Following [#2082](https://github.com/grafana/k6/pull/2082), k6's JS interpreter now has support for [Source Maps](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map). In the context of k6 scripts, it ensures that exceptions thrown from imported modules are reported with correct line information; as is described in [#1804](https://github.com/grafana/k6/issues/1804).
```

Comment:
```suggestion
Following [#2082](https://github.com/grafana/k6/pull/2082), k6's now has support for [Source Maps](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map). This means that it will now try to load sourcemaps from the filesystems or from inside the script, based on the standard-ish `//# sourceMappingURL=` comments.  Additionally as k6 internally uses babel to transform ES6+ scripts to ES5.1+, it will now use babel's ability to generate sourcemaps, including combining with previously generated ones, to report correct line numbers. This should fix [#1804](https://github.com/grafana/k6/issues/1804), but some problems are more tweaking are expected.
```

Reply:
Thanks a lot for this 🙇🏻 Greatly appreciated 😸

Reply:
Note that there's a double "are" now, one of which should probably be an "and":

> but some problems are more tweaking are expected.

should read "but some problems and more tweaking are expected."

(And I think that there should be a comma after "Additionally[,] as k6")

Reply:
Good catch! Thanks 🙇🏻

Reply:
Rephrased as "; however, we anticipate some more issues will arise, and further tweaking will be necessary"